cdbot/constants.py

```diff
@@ -129,6 +129,10 @@ class Exchange:
 README_SEND_ALIASES = ["create", "push", "generate", "send", "make", "build", "upload"]
 README_RECV_ALIASES = ["fetch", "get", "pull", "download", "retrieve", "dm", "dl"]
 
+# Readme auto update constants
+README_CHANNEL_ID = "409853512185282561"
```

Comment:
Can we make these overridable with environment variable like the other variables in `constants.py` please

Reply:
Like QUOTES_CHANNEL_ID?

Reply:
Yeah
```suggestion
README_CHANNEL_ID = int(environ.get("README_CHANNEL_ID", "409853512185282561"))
```

Reply:
Ok thanks